packages/browser-wallet/src/shared/utils/ccd.ts

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
Avoid duplicating code from the desktop wallet. If we need the utility functions in both projects, then we should make a shared library instead.